provider-ci/internal/pkg/templates/bridged-provider/Makefile

```
@@ -208,7 +208,7 @@ bin/pulumi-java-gen: .pulumi-java-gen.version
 # - Run make ci-mgmt to apply the change locally.
 #
 ci-mgmt: .ci-mgmt.yaml
-	rm -f .github/workflows/*.yml # Copied from update-workflows.yml
+	find pulumi-$(PACK)/.github/workflows/*.yml -type f ! -name "$(PACK)*.yml" -delete
```

Shouldn't this exclude the `pulumi-$(PACK)/` prefix?

```suggestion
	find .github/workflows/*.yml -type f ! -name "$(PACK)*.yml" -delete
```

---

You are right, good catch